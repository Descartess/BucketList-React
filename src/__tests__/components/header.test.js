import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../components/header';

describe('Header snapshot', () => {
  const props = {
    onShowAddItem: jest.fn(),
    add: jest.fn(),
    title: '',
  };
  it(' Header component renders correctly', () => {
    const rendered = renderer.create(
      <Header {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
