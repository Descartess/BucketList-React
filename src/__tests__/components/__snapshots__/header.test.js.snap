// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot  Header component renders correctly 1`] = `
<div
  className="panel-heading"
>
  
  <div
    className="pull-right"
    id="addBucket"
    onClick={[Function]}
  >
    <span
      className="glyphicon glyphicon-plus"
    />
     Add
  </div>
</div>
`;
